Packs/Cymulate/Integrations/Cymulate_v2/Cymulate_v2.py

```diff
@@ -230,27 +225,25 @@ def extract_status_commands_output(result: dict) -> dict:
         'id': result.get('id') if result.get('id') else result.get('_id'),
         'inProgress': result.get('inProgress'),
         'progress': result.get('progress'),
-        'addresses': result.get('addresses'),
         'categories': result.get('categories'),
+        'addresses': result.get('addresses'),
     }
-    return output
+    return remove_empty_elements(output)
 
 
-def build_data(params: dict) -> str:
-    """ Helper function to build HTTP request parameters. Cymulate's API does not accept Python's
-     'True' and 'False' objects, it requires a true and false objects (lowercase t and f).
-     In addition this function converts the dictionary to a string.
+def extract_template_output(raw_response: dict):
+    """Helper function to replace all the `_id` key to `id`.
 
     Args:
-        params (dict): parameter dictionary send to the API.
+        raw_response (dict): Raw response returned from the API.
 
     Returns:
-        str: params as a string, with lower case boolean values.
+        list[Dict]: The raw response where the `_id` key has replaced with `id`.
     """
-    data = json.dumps(params).replace('False', 'false')
-    data.replace('False', 'false')
-    data.replace('True', 'true')
-    return data
+    outputs = copy.deepcopy(raw_response)
+    for dictionary in outputs:
+        dictionary['id'] = dictionary.pop('_id')
```

Comment:
can we be certain that the object will contain the `_id` field? if not, use default to avoid exceptions
```suggestion
        dictionary['id'] = dictionary.pop('_id', None)
```

Reply:
Yes, we can. Nevertheless, I added the default just in case the API will be changed.